CMakeLists.txt

```diff
@@ -559,6 +559,7 @@ if((ENABLE_LOG) AND (CMAKE_SYSTEM_NAME STREQUAL "Linux"))
 
     set (install-3rd_party_logger_var "install-3rd_party_logger")
   endif()
+  set(LOG4CXX_LOGGER ON)
```

Comment:
@pestOO you did not get the point. `LOG4CXX_LOGGER` used not only for testing mode, it's help macro to check `log4cxx` logger using in current build and uses in all builds not depend on `BUILD_TESTS` flag. Anyway, root `CMakeLists.txt` file will be refactored in next PR and this peace of code will be changed